### PR TITLE
Classifier create gRPC

### PR DESCRIPTION
**Pull request by Sandro-Meireles**
Proto file PR: https://github.com/Ilhasoft/weni-protobuffers/pull/18
**Remember**, in order for this `Pull Request` to be approved, you must also approve that of [weni-protobuffers](https://github.com/Ilhasoft/weni-protobuffers/).

### CODE REVIEW

File: weni/grpc_central/serializers.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+        try:
+            return model.objects.get(**query)
+        except model.DoesNotExist:
+            cls.raises_not_fount(model.__name__, value)
+        except exceptions.ValidationError:
+            cls.raises_not_fount(model.__name__, value)
+
+    @classmethod
+    def raises_not_fount(cls, model_name, value):
```
Comment (jcbalmeida) [inline weni/grpc_central/serializers.py:32]: do you mean  _raises_not_found_?

Comment (Sandro-Meireles) [reply weni/grpc_central/serializers.py]: Yeah, thank you. I uploaded a commit with the adjustment.

File: weni/grpc_central/serializers.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    @classmethod
+    def _get_object(cls, model, value, query_parameter: str) -> models.Model:
+
+        query = {query_parameter: value}
+
+        try:
+            return model.objects.get(**query)
+        except model.DoesNotExist:
+            cls.raises_not_fount(model.__name__, value)
+        except exceptions.ValidationError:
```
Comment (jcbalmeida) [inline weni/grpc_central/serializers.py:28]: I think exceptions.ValidationError will not occur in get(). Instead, we should handle both model.DoesNotExist and  model.MultipleObjectsReturned.

File: weni/grpc_central/serializers.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+from django_grpc_framework import proto_serializers
+
+from temba.orgs.models import Org
+
+
+class SerializerUtils:
+    
+    @classmethod
+    def get_user_object(cls, value, query_parameter: str = "pk") -> User:
+        return cls._get_object(User, value, query_parameter)
```
Comment (jcbalmeida) [inline weni/grpc_central/serializers.py:13]: Why not use User.objects.get(**{query_parameter:value})?

File: weni/grpc_central/serializers.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    
+    @classmethod
+    def get_org_object(cls, value, query_parameter: str = "pk") -> Org:
```
Comment (jcbalmeida) [inline weni/grpc_central/serializers.py:16]: Same here, we could call Org.objects.get(**{query_parameter: value})

File: weni/grpc_central/serializers.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        return cls._get_object(Org, value, query_parameter)
+    
+    @classmethod
+    def _get_object(cls, model, value, query_parameter: str) -> models.Model:
```
Comment (jcbalmeida) [inline weni/grpc_central/serializers.py:20]: We can use simply _get_object(cls, model, **query)...

**Review by jcbalmeida — APPROVED** [review]

**Pull request merged**